### PR TITLE
Update the CODEOWNERS file.

### PR DESCRIPTION
**Pull request by SilasKenneth**

### CODE REVIEW

**Review by Ndiritu — APPROVED** [review]

**Comment by andrueastman** [comment]
The codeowner file contains errors with the change. I think this may not work after merge because the repo is in the microsoft org while the team is in the microsoftgraph org. We may have to create a team in the microsoft org similar to https://github.com/orgs/microsoft/teams/kiota-write

**Comment by SilasKenneth** [comment]
@andrueastman this is now updated.

**Pull request merged**
